### ISSUE TITLE
JQuery - Slick Slider - Only shows the first image

### ISSUE THREAD

**Issue by DoopsDesigns** [kenwheeler/slick#3654]
I have created a slick slider on a site. I can see it goes into 'slick mode', but doesnt show the number of slides I have told it too. They are all there and I can make them show by changing their 'left' and 'opacity' css values.

I won't be able to replicate this as I have used slick reliably forever and will work if I dumb it down.

The slider is within the blue section under the main image. There should be 4 slicked slides. However it only shows one.

If anyone thinks they can crack it pop over here: https://development.materialshub.com/2018/11/13/altered-visions/

====================================================================

Steps to reproduce the problem...
I have never had trouble with this slider before - its amazing...

====================================================================

What is the expected behaviour?
To show all four slides

====================================================================

What is observed behaviour?

1 slide shows at the correct width to show all four but the other slides are given opacity 0 and a left value that places them on top of each other.

====================================================================

More Details

- Which browsers/versions does it happen on? Chrome, Firefox, Safari
- Which jQuery/Slick version are you using? 1.8.1
- Did this work before? it is a new build

**Comment by DoopsDesigns** [comment]
:S - Nobody with any ideas?

**Comment by DoopsDesigns** [comment]
Please! My brain hurts!

**Comment by logangarcia** [comment]
Your development site is just a static image.
